test/ExecutionFramework.cpp

```diff
@@ -40,7 +40,7 @@ h256 const EmptyTrie("0x56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5
 
 string getIPCSocketPath()
 {
-	string ipcPath = dev::test::Options::get().ipcPath;
+	string ipcPath = dev::test::Options::get().ipcPath.c_str();
```

Comment:
Why convert to a c_str and back? Is there no option to convert to string directly?

Reply:
I couldn't find a better way at the time. But now I discovered that it is simply `.string()`